draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1711,6 +1711,78 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
 </section>
 </section>
 
+<section title="Trailer Fields" anchor="trailer.fields">
+   <iref item="trailer fields"/>
+   <iref item="trailers"/>
+
+<section title="Purpose" anchor="trailers.purpose">
+<t>
+   In some cases specific to the transfer coding or HTTP version, additional
+   metadata can be sent after the initial header section has been completed
+   (during or after transmission of the payload body) that might be
+   dynamically generated while the body is being sent, such as a message
```

Comment:
"that might be dynamically generated while the body is being sent" makes this sentence really wordy. Suggest deleting or moving it.

Reply:
deleted